packages/lisk-cryptography/test/nacl.js

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+// Require is used for stubbing
+const hash = require('../src/hash');
+const sign = require('../src/sign');
+const convert = require('../src/convert');
+const nacl = require('../src/nacl/nacl');
+
+const {
+	randombytes,
+	signKeyPair,
+	signDetached,
+	detachedVerify,
+	box,
+	boxOpen,
+} = nacl;
+
+const makeInvalid = str => {
+	const char = str[0] === '0' ? '1' : '0';
+	return `${char}${str.slice(1)}`;
+};
+
+describe('nacl', () => {
+	const CRYPTO_SIGN_PUBLICKEYBYTES = 32;
```

Comment:
maybe these one doesn't need to be capital since other constants are not capital?